src/devices/Ft232H/Ft232HI2c.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ft232H
+{
+    /// <summary>
+    /// I2C Device for FT232H
+    /// </summary>
+    public class Ft232HI2c : I2cDevice
+    {
+        private Ft232HI2cBus _i2cBus;
+        private int _deviceAddress;
+        private I2cConnectionSettings _settings;
+
+        internal Ft232HI2c(Ft232HI2cBus i2cBus, int deviceAddress)
+        {
+            _i2cBus = i2cBus;
+            _deviceAddress = deviceAddress;
+            _settings = new I2cConnectionSettings((int)i2cBus.DeviceInformation.LocId, deviceAddress);
+        }
+
+        /// <inheritdoc/>
+        public override I2cConnectionSettings ConnectionSettings => _settings;
+
+        /// <inheritdoc/>
+        public override void Read(Span<byte> buffer)
+        {
+            _i2cBus.Read(_deviceAddress, buffer);
+        }
+
+        /// <inheritdoc/>
+        public override byte ReadByte()
```

Comment:
isn't the default implementation gonna do the same? if so, I'd suggest to remove this. The only time you should override this if you can provide faster implementation than default (i.e. specific interop for reading a single byte)

Reply:
same comment for WriteByte